src/filesys.cpp

```diff
@@ -27,16 +27,18 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 #include "config.h"
 #include "porting.h"
+#include <algorithm> 
 
 namespace fs
 {
 
 #ifdef _WIN32 // WINDOWS
-
 #define _WIN32_WINNT 0x0501
 #include <windows.h>
 #include <shlwapi.h>
 
+static const std::string bad_names[] = { "CON", "PRN", "AUX", "NUL", "COM1", "COM2", "COM3", "COM4", "COM5", "COM6", "COM7", "COM8", "COM9", "LPT1", "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8", "LPT9", "" };
```

Comment:
that should definitely not be in this list, instead it should be checked in the appropriate function or even in the mainmenu lua code